patientsearch/src/js/App.js

```diff
@@ -2,7 +2,8 @@ import React, { Component } from "react";
 import CssBaseline from '@material-ui/core/CssBaseline';
 import { ThemeProvider } from '@material-ui/styles';
 import Header from './components/Header';
-import Search from './components/Search';
+//import Search from './components/Search';
```

Comment:
I'm a fan of keeping the code clean.  If you see value in keeping the commented out `Search` blocks, all good.  Otherwise, let's purge.